Gemfile.lock

```diff
@@ -93,7 +97,8 @@ GEM
       i18n (>= 1.6, < 2)
     faraday (1.0.1)
       multipart-post (>= 1.2, < 3)
-    ffi (1.13.1-x64-mingw32)
+    ffi (1.13.1)
```

Comment:
won't this break windows machines?

Reply:
ha, I really don't know what happened there. can we get someone on windows to run `bundle install` so that we compare the result?